lib/stripe/connect/account.ex

```diff
@@ -104,106 +110,97 @@ defmodule Stripe.Account do
           fields_needed: [String.t()]
         }
 
+  @type business_profile :: %{
+          mcc: String.t() | nil,
+          name: String.t() | nil,
+          product_description: String.t() | nil,
+          support_address: %{
+            city: String.t() | nil,
+            country: String.t() | nil,
+            line1: String.t() | nil,
+            line2: String.t() | nil,
+            postal_code: String.t() | nil,
+            state: String.t() | nil
+          },
+          support_email: String.t() | nil,
+          support_phone: String.t() | nil,
+          support_url: String.t() | nil,
+          url: String.t() | nil
+        }
+
+  @type capabilities :: %{
+          card_payments: String.t() | nil,
+          legacy_payments: String.t() | nil,
+          platform_payments: String.t() | nil
+        }
+
   @type t :: %__MODULE__{
           id: Stripe.id(),
           object: String.t(),
-          business_logo: String.t() | nil,
-          business_name: String.t() | nil,
-          business_url: String.t() | nil,
+          business_profile: business_profile | nil,
+          business_type: String.t() | nil,
+          capabilities: capabilities | nil,
           charges_enabled: boolean,
-          created: Stripe.timestamp() | nil,
+          company: company | nil,
           country: String.t(),
-          debit_negative_balances: boolean,
-          decline_charge_on: decline_charge_on,
+          created: Stripe.timestamp() | nil,
           default_currency: String.t(),
           details_submitted: boolean,
```

Comment:
The `deleted` key seems to be missing here

Reply:
Where do you see this? (just making sure we are both looking at the same documentation)

https://stripe.com/docs/api/accounts/object
https://raw.githubusercontent.com/stripe/openapi/master/openapi/spec3.yaml

Reply:
Hmm, I ran the tests against the stripe-mock and got this warning
```[warn]  Extra keys were received but ignored when converting Stripe.Account: [:deleted]```

But it seems the `deleted` is only added to the response in the test for the `Account.delete/2` function.

Reply:
Seems `Account.delete/2` returns the deleted account but that is no more than this in JSON
```
{
  "id": "acct_1032D82eZvKYlo2C",
  "object": "account",
  "deleted": true
}
```

We may need a different pattern to deal with deletes of this kind.